decidim-admin/app/views/decidim/admin/organization/_form.html.erb

```diff
@@ -74,6 +74,28 @@
 
 <div class="row">
   <div class="columns xlarge-6">
+    <%= form.file_field :official_img_header %>
+    <% if current_organization.official_img_header.present? %>
+      <label><%= t('.current_image') %></label>
+      <%= image_tag current_organization.official_img_header.url %>
```

Comment:
Versions:

- homepage image: version "big", 1920px wide
- logo: version "medium", 600px wide
- favicon: big: 152px, medium: 64px, small: 32px wide
- official_img_header and official_img_footer: no versions, they `resize_to_limit`. Header: 160px wide, footer: 600px wide (?)

What would you like to have? 😕